utils/oscap-info.c

```diff
@@ -57,11 +57,23 @@ struct oscap_module OSCAP_INFO_MODULE = {
     .parent = &OSCAP_ROOT_MODULE,
     .summary = "info module",
     .usage = "some-file.xml",
-    .help = "Print information about a file",
+    .help = "Print information about a file\n"
+    "\n"
+    "Options:\n"
+    "   --fetch-remote-resources \r\t\t\t\t - Download remote content referenced by DataStream.\n",
     .opt_parser = getopt_info,
     .func = app_info
 };
 
+static void _download_reporting_callback(bool warning, const char *format, ...)
+{
+	va_list argptr;
+	va_start(argptr, format);
+	vfprintf(stderr, format, argptr);
```

Comment:
Why does the download callback for `oscap info` write on stderr? The same messages in `xccdf eval` are written on stdoud. I think it should be consistent, because the text of message is same and has same meaning.